_posts/2019/2019-05-21-making-a-move-abroad-from-bolivia-to-hamburg.md

```diff
@@ -0,0 +1,58 @@
+---
+layout: post
+title: Making a move abroad - from Bolivia to Hamburg
```

Comment:
This is just a suggestion. Maybe this can even get more emotional, depending on your reasons to move away from Bolivia.